autocertkit/networkconf.example

```diff
@@ -12,6 +12,11 @@ eth3 = 1,[200]
 
 # static_<network _id>_<vlan_id> = <ip_start>,<ip_end>,<netmask>,<gw>
 
+# In case, management network is not having DHCP and management interface
+# is not specified above ( management interface is not being certified),
+# It is mandatory to specify static addressing, i.e.:
+#  static_management = 192.168.0.2,192.168.0.10,255.255.255.0,192.168.0.1
```

Comment:
Hi @bpwook ,

static_0_0 refers to static ip range for network 0, vlan 0.
Now if management interface is not being tested by ACK (ie  is not mentioned as iterface in netconf), then we have no way to tell what static ip addressing should be used by ack for vms as they always have their 1st vif on management.
The ip range static_management is meant for VMs.
This lead to partner issue "MLX-27", we have detailed discussion of it over there.